root/tag/DeleteTag.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import FormSubmit from '../components/FormSubmit';
+
+import TagLayout from './TagLayout';
+
+type Props = {
+  +$c: CatalystContextT,
+  +deleteDownvoted?: boolean,
+  +tag: TagT,
+};
+
+const DeleteTag = ({
+  $c,
+  deleteDownvoted = false,
+  tag,
+}: Props): React.Element<typeof TagLayout> | null => {
+  const title = deleteDownvoted
+    ? l('Delete tag downvotes')
+    : l('Delete tag uses');
```

Comment:
elsewhere we use "delete my upvotes," or "uses (upvotes)".  we should probably use upvotes consistently to avoid confusion - what do you think?

Reply:
True. I went with uses (upvotes) everywhere for now.

Reply:
Would it be alright to just use _tag upvotes_ and avoid introducing a new unneeded verbiage with _tag uses_?

_upvote_ is used everywhere already, [documentation](https://musicbrainz.org/search?type=doc&query=upvote#gsc.tab=0&gsc.q=upvote&gsc.page=1) included. I didn’t find any occurrence of _use_ for tags so far.

Reply:
Ok, I'll make the change. It feels a bit confusing in that as a user I'd expect "upvote" to be when I click to agree with an existing tag, but not when I enter a tag by writing it, but it's entirely true that's a much more general issue than here so we should be consistent for now.

Reply:
I changed these. I also added `folksonomy` contexts where relevant and changed some "votes against" to "downvotes of" for parity.